extensions/hdrhistogram/src/main/java/org/radargun/stats/HistogramOperationStats.java

```diff
@@ -250,4 +252,12 @@ private void readObject(ObjectInputStream s) throws IOException, ClassNotFoundEx
       }
       compacted = new Histogram(ranges, counts);
    }
+
+   private void recordValue(long startTimeNs, long endTimeNs, long intendedTimeNs) {
+      if (intendedTimeNs != 0) { // if intended time is set we can compute response/wait
+         histogram.recordValue(endTimeNs - intendedTimeNs);
```

Comment:
If intendedTimeNs is equals 5 and endTimeNs is equals 4.
What should we do in this case?

Reply:
How could the response be received before you intended to send it? If you intend to send the message later, you simply don't send it yet (regrettably it's not as easy when you're already past the time :)), so if this happens it's a bug.